.eslintrc.yaml

```diff
@@ -0,0 +1,131 @@
+root: true
```

Comment:
I noticed that this file is `yaml` while some others configuration files are `js`. Shouldn't we keep one file type for consistency?

Reply:
The YAML files is the application configuration and should be as good as empty in the ideal world, the js files are part of the `eslint-plugin-amsterdam` which could eventually live somewhere out of this repository.

Reply:
Not sure I understand the explanation, why don't we keep the same format for eslint configuration? It doesn't really matter which format as long is consistent overall. I would choose js because most of the projects at `datapunt` are using this format. As a matter of fact, this is the original format before this PR.

Reply:
I'm also still interested to hear your reasoning for choosing the yaml format. :)

Reply:
@boris-arkenaar, mee too. @jpoppe, we agreed to talk about this PR next week. I think this might be merged by mistake. Could you please revert this?